2024-05-07.mdx

```diff
@@ -0,0 +1,43 @@
+---
+date: 5-7-2024
+---
+
+### What's new
+
+- We continued to improve the Postgres wire compatibility with more clients and ORMs:
+    - Improved the extended wire protocol by supporting flows not starting with a `Sync` message. 
+    - Added support for the `DEALLOCATE` statement.
+    - Added support for `current_user`, `current_role`, `session_user`, and `user` functions.
+    - Added support for `GREATEST` and `LEAST` functions.
+    - Correctly handle `CommandComplete` and `EmptyQueryResponse` messages in the Postgres wire protocol. This fixes a bug where the client would hang in particular situations.
+    - Added support for `information_schema._pg_expandarray` over the Postgres wire protocol.
+    - Clarified error message on failed internal authentication.
+- We also continued to improve Dedicated Clusters, currently in private alpha: 
+    - It's now possible to enable extensions (`CREATE EXTENSION`).
+    - Termination is now self-service.
```

Comment:
The BE part for the termination is done, i don't think the FE is ready, or at least i could find no "terminate" button